app.js

```diff
@@ -21,6 +22,13 @@ setupMiddleware(app);
 // registerRoutes(app);
 app.use(routes);
 
+// Set templating engines
+app.set('view engine', 'html'); // set default engine as "html"
+app.set('views', path.join(__dirname, './app/views')); // set path
```

Comment:
What's the purpose of this line of code?

Reply:
This is used to tell express where can find the template files.